src/components/Proceso.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import './Styles/Proceso.css' 
+
+const Proceso = () => {
+
+    return (
+        <> 
+            <div className="container-fluid About">
+                <header className="section-header pt-5">
+                    <h2>Proceso</h2>
+                </header>
+                <div className="container timeline mb-5">
+                    <div className="containerAbout containerAbout-left">
+                        <div className="image box-white"><h1 className="text-center pt-2 text-white">1</h1></div>
+                        <div className="content black-box">
+                            <h3 className="aboutTitle text-center"><i className="fas fa-users me-3"></i>Envio de solicitud</h3>
+                            <p  className="text-center">
+                                Et hinc magna voluptatum usu, cum veniam graece et. Ius ea scripserit temporibus, pri cu harum tacimates neglegentur. At adipisci incorrupte nam. Cu qui sumo appareat constituto.
+                            </p>
+                        </div>
+                    </div>
```

Comment:
Potencial componente.
```suggestion
    const SuggestedComponent = ({ containerPlace, containerColor, number, iconCSS, title, content }) => {
        const invertColor;
        if (contentColor == "white"){
            invertColor = "black";
        } else {
            invertColor = "white"
        };
    
        return(
            <div className=`containerAbout containerAbout-${ containerPlace }`>
                <div className=`image box-${ containerColor }`><h1 className=`text-center pt-2 text-${ containerColor }`>{ value }</h1></div>
                <div className=`content { invertColor }-box`>
                    <h3 className="aboutTitle text-center"><i className=`${ iconCSS } me-3`></i>{ title }</h3>
                    <p  className="text-center">{ content }</p>
                </div>
            </div>
        );
    };
    
    SuggestedComponent.propTypes = {
        containerPlace: PropTypes.oneOf(['left', 'right']),
        containerColor: PropTypes.oneOf(['white', 'black']),
        number: PropTypes.number,
        iconCSS: PropTypes.string,
        title: PropTypes.string,
        content: PropTypes.string
    };
```

Reply:
```suggestion
const white = "white";
const black = "black";

const TimelineItem = ({ containerPlace, containerColor, number, iconCSS, title, content }) => {
	const invertColor = contentColor == white ? white: black;

	return(
		<div className=`containerAbout containerAbout-${ containerPlace }`>
			<div className=`image box-${ containerColor }`>
				<h1 className=`text-center pt-2 text-${ containerColor }`>{ value }</h1>
			</div>
			<div className=`content ${ invertColor }-box`>
				<h3 className="aboutTitle text-center"><i className=`${ iconCSS } me-3`></i>{ title }</h3>
				<p  className="text-center">{ content }</p>
			</div>
		</div>
	);
};

TimelineItem.propTypes = {
	containerPlace: PropTypes.oneOf(['left', 'right']),
	containerColor: PropTypes.oneOf(['white', 'black']),
	number: PropTypes.number,
	iconCSS: PropTypes.string,
	title: PropTypes.string,
	content: PropTypes.string
};
```

Reply:
```suggestion
const Timeline = ({ listOfItems }) => {
	const timelineItems = [];
	listOfItems.forEach((item, index) => {
		const index_is_even = index % 2 === 0;
		timelineItems.push({
			containerPlace: index_is_even ? 'left': 'right',
			containerColor: index_is_even ? 'white': 'black',
			number: index + 1,
			iconCSS: item.iconCSS,
			title: item.title,
			content: item.content
		});
	});
	return (
		<div className="container timeline mb-5">
			listOfItems.map(TimelineItem);
		</div>
	);
}
```